massbalancemachine/data_processing/Dataset.py

```diff
@@ -82,7 +87,7 @@ def convert_to_monthly(
         *,
         vois_climate: "list[str]",
         vois_topographical: "list[str]",
-        meta_data_columns:"list[str]"=["RGIId", "POINT_ID", "ID", "N_MONTHS", "MONTHS"]
```

Comment:
shouldn't the default be the meta data columns from the config file?

Reply:
Otherwise looks fine to me :)

Reply:
Later in this method, the meta_data will be still set from the config file.